source/localizable/models/relationships.md

```diff
@@ -43,7 +43,7 @@ import Model from 'ember-data/model';
 import { belongsTo } from 'ember-data/relationships';
 
 export default Model.extend({
-  post: belongsTo('post')
+  blogPost: belongsTo('blog-post')
```

Comment:
That makes sense! I will push up an update that changes `post` to `blog-post` throughout the whole page.

Thanks for the feedback guys!